core/trino-main/src/test/java/io/trino/operator/scalar/timestamptz/TestTimestampWithTimeZone.java

```diff
@@ -1987,6 +1987,14 @@ public void testToIso8601()
                 .hasType(createVarcharType(41))
                 .isEqualTo("2020-05-01T12:34:56.123456789012+05:45");
 
+        assertThat(assertions.expression("to_iso8601(TIMESTAMP '2020-11-01 07:00:00.123456789012 UTC' AT TIME ZONE 'America/Chicago')"))
+                .hasType(createVarcharType(41))
+                .isEqualTo("2020-11-01T01:00:00.123456789012-06:00");
+
+        assertThat(assertions.expression("to_iso8601(TIMESTAMP '2020-11-01 07:00 UTC' AT TIME ZONE 'America/Chicago')"))
+                .hasType(createVarcharType(28))
+                .isEqualTo("2020-11-01T01:00:00-06:00");
+
```

Comment:
Can you add tests for all precisions? See examples above. Also, please add a comment before all those tests explaining that this tests timestamps that fall in the DST gap.

Reply:
Thanks for your comments! Updated the test cases.